tests/end_to_end/test_helper.cpp

```diff
@@ -250,7 +250,7 @@ _preprocess_load_native_module(_Inout_ service_context_t* context)
     _is_platform_preemptible = !_is_platform_preemptible;
 
     context->dll = LoadLibraryW(context->file_path.c_str());
-    REQUIRE(((context->dll != nullptr) || (_expect_native_module_load_failures)));
+    REQUIRE(((context->dll != nullptr) || (get_native_module_failures())));
```

Comment:
```suggestion
    REQUIRE(((context->dll != nullptr) || get_native_module_failures()));
```
nit for consistency with line 233